### DISCUSSION TITLE
Improvement for contributors page

### DISCUSSION THREAD

**Discussion by EricGao888** [community/community#19662] [Other Feature Feedback, Questions, & Ideas]
- Currently, it seems there are some errors when calculating contributions in `contributors` page. Some contributions somehow not counted. I've found this issue in several apache projects repo.
- Add a new feature: enable repo maintainers to count contributions in affiliated repo to the main repo.

**Comment by JhonnyLusonode** [comment]
@EricGao888 , if you belive it´s a error/bug

Please report it to [Github Bug Report](https://support.github.com/contact/bug-report).

Hope this clarifies.

**Reply by EricGao888** [reply]
Thanks for the reply.
